uwuno/src/main/java/com/learning/uwuno/util/utils.java

```diff
@@ -0,0 +1,61 @@
+package com.learning.uwuno.util;
```

Comment:
Changed name to match the other file names and I was too lazy to type more letters. :D